### PR TITLE
chore(release): pull release/v1.39.0 into main

### PR DESCRIPTION
**Pull request by devops-github-rudderstack**
:crown: *An automated PR*

### CODE REVIEW

File: src/v0/destinations/clevertap/transform.js [hunk 1]
```diff
@@ -377,9 +383,48 @@ const processEvent = (message, destination) => {
...
+      if (!event?.message?.statusCode) {
+        // already transformed event
+        resp = process(event);
+      }
+      eventsChunk.push({
+        message: Array.isArray(resp) ? [...resp] : resp,
+        metadata: event.metadata,
+        destination: event.destination,
+      });
+    } catch (error) {
```
Comment (krishna2020) [inline src/v0/destinations/clevertap/transform.js:410]: @yashasvibajpai take backlog item to cover failure scenario test cases too

Comment (yashasvibajpai) [reply src/v0/destinations/clevertap/transform.js:410]: Added for myself

File: src/v0/destinations/june/data/JuneGroupConfig.json [hunk 2]
```diff
@@ -28,7 +28,23 @@
...
+    "required": false,
+    "metadata": {
+      "excludes": [
+        "page",
+        "traits",
+        "sessionId",
+        "sessionStart",
+        "consentManagement"
+      ]
     }
```
Comment (krishna2020) [inline src/v0/destinations/june/data/JuneGroupConfig.json:48]: @mihir-4116 @aashishmalik seems like some issue with back-merge PR - review the June/data config changes & raise PR to release branch to fix these files

**Review by mihir-4116 — APPROVED** [review]

File: src/v0/destinations/june/data/JuneGroupConfig.json (hunk 2 above)
Comment (mihir-4116) [reply src/v0/destinations/june/data/JuneGroupConfig.json:48]: Removed duplicate context from June Group Config

**Review by krishna2020 — APPROVED** [review]

**Review by sandeepdsvs — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**